platform/openide.filesystems/src/org/openide/filesystems/MultiFileObject.java

```diff
@@ -205,7 +205,6 @@ private void update() {
         }
 
         Iterator it = now.iterator();
-
         while (it.hasNext()) {
             FileObject fo = (FileObject) it.next();
```

Comment:
What happened with this? Can't this converted into for as well?

Reply:
Unfortunately it not "low hanging fruit".. When I change this to the foreach pattern, I get the following error message..

```
   [repeat] /home/bwalker/netbeans-11.0/platform/openide.filesystems/src/org/openide/filesystems/MultiFileObject.java:207: error: incompatible types: Object cannot be converted to FileObject
   [repeat]         for (FileObject fo : now) {
   [repeat] 
```                             ^
So I left it out..

My goal is simply to get the build warnings down..